lib/ims/lti/models/messages/message.rb

```diff
@@ -165,6 +165,26 @@ def method_missing(meth, *args, &block)
       end
     end
 
+    def to_jwt(private_key:, originating_domain:, algorithm: :HS256)
+      ims = post_params
```

Comment:
Is this assignment really necessary? Also, where can I look at the spec to see how the payload should be formatted?